### ISSUE TITLE
UNUSED_LOCAL_VARIABLE false positive with var used in hashes

### ISSUE THREAD

**Issue by Vintic** [cflint/CFLint#667]
Unused variable on line 3
```
<cfcomponent displayname="myComponent">
    <cffunction name="getData">
        <cfset var testVar = 123>

        <cfsavecontent variable = "sql_query">
            <cfoutput>
                SELECT #testVar#
            </cfoutput>
        </cfsavecontent>
    </cffunction>
</cfcomponent>

```

**Comment by Vintic** [comment]
if cfoutput contains any text, before, or after #testVar#, bug is reported

**Comment by Vintic** [comment]
SyntaxError: Line1:10 no viable alternative at input '#testVar#asd'
